cfgov/v1/atomic_elements/atoms.py

```diff
@@ -112,6 +112,33 @@ class Button(Hyperlink):
     ], default='regular')
 
 
+class ImageBasicStructValue(blocks.StructValue):
+    @property
+    def url(self):
+        upload = self.get('upload')
+
+        if upload:
+            return upload.get_rendition('original').url
+
+    @property
+    def alt_text(self):
+        # TODO: This duplicates the logic in v1.jinja2tags.image_alt_value,
+        # which cannot be called here because of a circular import. It would
+        # be better to deprecate the image_alt_value tag in favor of using
+        # this logic wherever we use ImageBasic atoms.
```

Comment:
For renditions there's no need for `image_alt_value` since we can already rely on [`rendition.alt`](https://github.com/cfpb/cfgov-refresh/blob/7c95ee016c20480a6085031ce9685fb063309cb8/cfgov/v1/models/images.py#L109), which calls [`CFGOVImage.alt`](https://github.com/cfpb/cfgov-refresh/blob/7c95ee016c20480a6085031ce9685fb063309cb8/cfgov/v1/models/images.py#L109), which returns the right thing or the empty string if left empty. See other comment below.

Reply:
If that's the case, I'm not 100% sure why we needed `image_alt_value` in the first place. Feels like we need to revisit/reboot our image handling, in general.